lib/install/config/webpacker.yml

```diff
@@ -1,21 +1,35 @@
 # Note: You must restart bin/webpack-dev-server for changes to take effect
 
 default: &default
+  # Values used by Ruby part of rails/webpacker
```

Comment:
I was confused by this comment; as I understand it, `source_path`, `public_root_path`, `public_output_path`, and `cache_path` (listed below)) are also consumed by the `@rails/webpacker` package as part of the default webpack config.

Reply:
Good point. I clarified that. I wanted to point out the subset of values that are used by the Ruby on Rails code.